src/github/client/github-installation-client.ts

```diff
@@ -252,27 +248,22 @@ export class GitHubInstallationClient extends GitHubClient {
 	/**
 	 * Attempt to get the commits page, if failing try again omiting the changedFiles field
 	 */
-	public async getCommitsPage(owner: string, repoName: string, perPage?: number, cursor?: string | number): Promise<getCommitsResponse> {
-		const response = await this.graphql<getCommitsResponse>(getCommitsQueryWithChangedFiles,
-			{
-				owner,
-				repo: repoName,
-				per_page: perPage,
-				cursor
-			}).catch((err) => {
-
-			if (!isChangedFilesError(err)) {
-				return Promise.reject(err);
-			}
-			this.logger.warn("retrying commit graphql query without changedFiles");
-			return this.graphql<getCommitsResponse>(getCommitsQueryWithoutChangedFiles,
-				{
-					owner,
-					repo: repoName,
-					per_page: perPage,
-					cursor
-				});
-		});
+	public async getCommitsPage(owner: string, repoName: string, perPage?: number, timeCutoff?: number, cursor?: string | number): Promise<getCommitsResponse> {
+		const variables = {
+			owner,
+			repo: repoName,
+			per_page: perPage,
+			cursor,
+			commitSince: timeCutoff ? new Date(Date.now() - timeCutoff).toISOString() : undefined
```

Comment:
feature flag?

Reply:
Or maybe not to feature-flag but no to include the key to the object if timeCutoffMsecs is not provided.

Reply:
The FF is the one that populate the variable.  No FF, variable is undefined and never set